### ISSUE TITLE
vscode intellisense doesn't work as it should on js (reopenning)

### ISSUE THREAD

**Issue by kaankoken** [Microsoft/vscode#67598]
First of all, this time try to not close my post because the great community that one of the mods mentioned is not helping on StackOverflow. They are just downvoting. Apart from this post, I opened 2 different post, and I got the same result, no answer & help.

I have written two same projects for the same purposes with the same code. However, one of them was written on Windows 10 in 2018, and the other one was on Arch Linux in 2019. Also, I opened them on Arch Linux. (Actually, I really do not the operating system and time matter, I just want it to inform everyone.)

Something bothers me about VScode. As it now and before, I wrote a RESTful API. The code is working perfectly. My problem is that Intellisense does not work completely on the newest one which is in 2019 on Arch Linux. However, It is working on 2018th one. I really do not understand that. Any thoughts?

_Example Code_
![image](https://user-images.githubusercontent.com/28218396/52020736-82236c80-2503-11e9-81d8-ae9846cf25c6.png)

_Screen Shots_
_(2018, Windows 10)_
![image](https://user-images.githubusercontent.com/28218396/52020834-cadb2580-2503-11e9-8803-6761ff5272f5.png)

_(2019, Arch Linux)_
![image](https://user-images.githubusercontent.com/28218396/52020868-e80ff400-2503-11e9-86e5-56a30f337b5d.png)

**Comment by mjbvz** [comment]
This isn't a support forum. That's why I directed you to stackoverflow.

Try following  [this guide](https://github.com/Microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions)  to report issues in a more actionable way. Start by including minimal example code or a minimal example project

**Comment by kaankoken** [comment]
@mjbvz I decided to open with `vscode-insiders`, and the codes that I wrote both of them are working. It is weird. I think, there could be a bug or glitch. In case, it is needed. These are the specs of `vscode` and `insiders`.
_code_
Version: 1.30.2
Commit: 61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8
Date: 2019-01-07T22:49:48.319Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
OS: Linux x64 4.20.4-arch1-1-ARCH

_insiders_
Version: 1.31.0-insider
Commit: e02e44464c02d617b98d0a3265bf5efef7794436
Date: 2018-12-21T06:15:51.434Z
Electron: 3.0.10
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.20.4-arch1-1-ARCH

Also, like 3 months ago or so, I had different problem. I could run c with `insiders`, but not with the normal version. After updates, the c running problem is gone.
